docs/company/getting-started.md

```diff
@@ -0,0 +1,35 @@
+# Getting Started at Thoth Tech
+
+Welcome to Thoth Tech!
+
+If you are reading this Getting Started Guide, then you have just enrolled in the Thoth Tech Company and we welcome you! 
+
+This guide is a place to introduce you to the company and all the resources availible to help get you onboarded, introduce you to the comapny and products, and skilled up to tackle the Trimester ahead.
+
+##Who are we?
+
+The best place to start to familiarise yourself with Thoth Tech is the [company charter](docs/company/../../charter.md). This is an important guide for understanding the mission and values of the company which informs the way that the company runs and is crucial to ensuring your experience is the best it can be.
+
+##What are our products?
+
+Thoth Tech currently maintains and develops three external products. [OnTrack](docs/product/ontrack/ontrack.md), [SplashKit](docs//../../products/splashkit/splashkit.md) and [DreamBig](docs/product/dreambig/dreambig.md). Thoth Tech also has one internal product that is being worked on currently, called Internal Systems.
+
+##Projects
+
+Before you complete your onboarding form, it is recommended that you have a look at the final [Company Report](ENTER LINK HERE) from T1 2022. This will give you a good sense of what projects are currently being worked on, what skils they require, and what is needing to be completed this trimester. Please note, there may be new projects availible to be started this trimester.
```

Comment:
Slight reword and typos:

```suggestion
Before you complete your onboarding form, it is recommended that you have a look at the final [Company Report](ENTER LINK HERE) from T1 2022. This will give you context around the projects that are currently in development, the skills required, and what is to be complete this trimester. _Please note:_ this will not include new projects available this trimester.
```